### PR TITLE
pcaudiolib: Update to 1.2

### PR DESCRIPTION
**Pull request by u2615**
**Summary**

- Full release note [1.2](https://raw.githubusercontent.com/espeak-ng/pcaudiolib/1.2/CHANGELOG.md)
- Changed source to official `espeak-ng` repo
- Removed patch not needed anymore to build
- Added `homepage` key to `package.yml` (part of getsolus/packages#411)
- Added `monitoring.yml` file (no CPE name found)

**Test Plan**

- Built package, Installed it in Local repo
- Rebuilt `espeak-ng` against it, installed and used it: no notable
  difference

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/p/pcaudiolib/package.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 name       : pcaudiolib
-version    : 1.0
-release    : 5
+version    : '1.2'
+release    : 6
 source     :
-    - https://github.com/rhdunn/pcaudiolib/archive/1.0.tar.gz : df2d0eb5de856dbdb7ffb4eb460dd65ea35ccdedc314854df469bc1ac4ad2c05
+    - https://github.com/espeak-ng/pcaudiolib/releases/download/1.2/pcaudiolib-1.2.tar.gz : 6fae11e87425482acbb12c4e001282d329be097074573060f893349255d3664b
+homepage   : https://github.com/rhdunn/pcaudiolib
```
Comment (malfisya) [inline packages/p/pcaudiolib/package.yml:6]: If we are using the official source, the homepage should match that

File: packages/p/pcaudiolib/package.yml [hunk 2]
```diff
@@ -12,7 +13,6 @@ builddeps  :
     - pkgconfig(alsa)
     - pkgconfig(libpulse-simple)
 setup      : |
-    %patch -p1 -i $pkgfiles/build-fixes.patch
     %reconfigure --with-pulseaudio --with-alsa --disable-static
```
Comment (malfisya) [inline packages/p/pcaudiolib/package.yml:16]: Please switch it `%configure`, it builds just fine with it.

**Review by malfisya — CHANGES_REQUESTED** [review]
Please add changelog to your commit message. More or less like this ;
`Full changelog [here](https://raw.githubusercontent.com/espeak-ng/pcaudiolib/1.2/CHANGELOG.md)`

**Comment by u2615** [comment]
Ok, thx for your patience and sorry about the miss,

- Changed title
- Corrected homepage
- Switched to `%configure`
- Added changelog

**Review by malfisya — APPROVED** [review]
I will merge this as is. I have one nitpick: The release note should be part of the summary section, preferably at the top. Cheers !

**Pull request merged**